### ISSUE TITLE
On Windows, there is a build folder in the zip package packaged by code-push-go, resulting in an update error

### ISSUE THREAD

**Issue by skylarklxlong** [htdcx/code-push-go#9]
error log
```
[CodePush] Update is invalid - A JS bundle file named "index.android.bundle" could not be found within the downloaded contents. Please check that you are releasing your CodePush updates using the exact same JS bundle file name that was shipped with your app's binary.
```
error image
![image](https://github.com/user-attachments/assets/184b0caa-c804-4df6-92bf-002782237223)

code-push-go build
```
set GOARCH=amd64
set GOOS=linux #or windows not available
go build -o code-push-go.exe main.go
mv code-push-go(.exe) <You project>
```

code-push-go create_bundle
```
./code-push-go create_bundle -t 1.1.1 -n xxx-Android -d TEST --description \"修复已知问题；优化使用体验；\"
```

**Comment by skylarklxlong** [comment]
On MacOS is OK ! 
![image](https://github.com/user-attachments/assets/293aefd7-b5aa-4044-9635-377a240d9e06)

**Comment by htdcx** [comment]
Try v1.0.3

**Comment by skylarklxlong** [comment]
v1.0.4 still doesn't work.

![image](https://github.com/user-attachments/assets/6889065e-9262-4b02-a191-a82525a45cb7)

![image](https://github.com/user-attachments/assets/0a4166fc-c5ef-4ec6-8a9a-35439b10206f)

**Comment by htdcx** [comment]
Fix to v1.0.6

**Comment by skylarklxlong** [comment]
> Fix to v1.0.6

It's ready for work. Thanks
